pkgs/applications/misc/digitalbitbox/default.nix

```diff
@@ -0,0 +1,109 @@
+{
+  stdenv,
+  autoconf,
+  automake,
+  curl,
+  fetchFromGitHub,
+  gcc5,
+  git,
+  libevent,
+  libtool,
+  libqrencode,
+  libudev,
+  libusb,
+  makeWrapper,
+  pkgconfig,
+  qt59,
+  withDebug ? false
```

Comment:
Is having this option worth it? Why make it configurable explicitly?

Reply:
Stripped the option out with 0c5f3d6972aded47db856108274f950da4047b18